erpnext/setup/doctype/holiday_list/holiday_list.js

```diff
@@ -6,13 +6,39 @@ frappe.ui.form.on("Holiday List", {
 		if (frm.doc.holidays) {
 			frm.set_value("total_holidays", frm.doc.holidays.length);
 		}
+
+		frm.call("get_supported_countries").then(r => {
+			frm.subdivisions_by_country = r.message;
+			frm.set_df_property("country", "options", Object.keys(r.message));
```

Comment:
Keep the first option blank in case someone wants to reset it? I think using Autocomplete will fix that.